package.json

```diff
@@ -38,5 +36,9 @@
     "src",
     "index.mjs",
     "index.js"
-  ]
+  ],
+  "dependencies": {
+    "@rollup/plugin-commonjs": "^19.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.0"
```

Comment:
Why did you move these to dependencies? These are needed for bundling and publishing the package, not for users of it.
Since this change seems completely unrelated I suggest to just revert these and only add the `index.d.ts` change.

Reply:
ah. didn't notice that. will submit another PR